engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -353,9 +353,12 @@ namespace dmGraphics
         TYPE_SAMPLER_2D       = 9,
         TYPE_SAMPLER_CUBE     = 10,
         TYPE_SAMPLER_2D_ARRAY = 11,
+        TYPE_FLOAT_VEC2       = 12,
+        TYPE_FLOAT_VEC3       = 13,
+        TYPE_FLOAT_MAT2       = 14,
+        TYPE_FLOAT_MAT3       = 15,
```

Comment:
We need to calculate things based on shader reflection (which is done in runtime for GL), and we use dmGraphics::Type everywhere so might as well add them here.